### PR TITLE
Change `topic_name` variable to `output_topic_name`

### LINKED ISSUE #28: Change `topic_name` variable to `output_topic_name`
Currently, in the config.yaml we define the default topic names like this:
```yaml
topic_name_config:
  default_error_topic_name: "${pipeline_name}-${component_name}-dead-letter-topic"
  default_output_topic_name: "${pipeline_name}-${component_name}-topic"
```
we use the `${topic_name}` in the `pipeline.yaml`. This internal variable is replaced by the `default_output_topic_name`. The naming of this variable is misleading and should be changed to `${output_topic_name}`.
Like the `${error_topic_name}` variable.

### PR DESCRIPTION
**Pull request by MichaelKora**
Fixes #28

### CODE REVIEW

File: docs/docs/user/examples/atm-fraud-pipeline.md [hunk 1]
```diff
@@ -83,20 +85,21 @@ kubectl port-forward --namespace kpops service/k8kafka-cp-kafka-connect 8083:808
     --config ./examples/bakdata/atm-fraud-detection/config.yaml \
     --execute
     ```
-   
+
 !!! Note
-    You can use the `--dry-run` flag instead of the `--execute` flag and check the logs if your pipeline will be
```
Comment (philipp94831) [inline docs/docs/user/examples/atm-fraud-pipeline.md:88]: @raminqaf Do you know if this indent is important?

Comment (raminqaf) [reply docs/docs/user/examples/atm-fraud-pipeline.md:88]: The indent is important we should not delete it

File: docs/docs/user/examples/atm-fraud-pipeline.md [hunk 2]
```diff
@@ -83,20 +85,21 @@ kubectl port-forward --namespace kpops service/k8kafka-cp-kafka-connect 8083:808
...
 
-You can use the [Streams Explorer](https://github.com/bakdata/streams-explorer) to see the deployed pipeline. 
+You can use the [Streams Explorer](https://github.com/bakdata/streams-explorer) to see the deployed pipeline.
 To do so, port-forward the service in a separate terminal session using the command below:
 
 ```shell
 kubectl port-forward -n kpops service/streams-explorer 8080:8080
 ```
-After that open [http://localhost:8080](http://localhost:8080) in your browser. 
+
```
Comment (raminqaf) [inline docs/docs/user/examples/atm-fraud-pipeline.md:101]: remove line

File: docs/docs/user/examples/atm-fraud-pipeline.md [hunk 3]
```diff
@@ -105,9 +108,9 @@ You should be able to see pipeline shown in the image below:
 </figure>
 
 !!! Attention
-    Kafka Connect needs some time to set up the connector. 
-    Moreover, Streams Explorer needs a while to scrape the information from Kafka connect.
-    Therefore, it might take a bit until you see the whole graph.
+Kafka Connect needs some time to set up the connector.
+Moreover, Streams Explorer needs a while to scrape the information from Kafka connect.
+Therefore, it might take a bit until you see the whole graph.
```
Comment (raminqaf) [inline docs/docs/user/examples/atm-fraud-pipeline.md:113]: Please revert all the indents here

File: docs/docs/user/examples/atm-fraud-pipeline.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
 # ATM fraud detection pipeline
 
-ATM fraud is a demo pipeline for ATM fraud detection. 
-The original by Confluent is written in KSQL 
-and outlined in this [blogpost](https://www.confluent.io/blog/atm-fraud-detection-apache-kafka-ksql/). 
-The one used in this example is re-built from scratch using [bakdata](https://bakdata.com/)'s 
+ATM fraud is a demo pipeline for ATM fraud detection.
```
Comment (raminqaf) [inline docs/docs/user/examples/atm-fraud-pipeline.md:3]: I don't see the point of the documentation changes in this PR please remove these changes from this PR.

**Review by raminqaf — CHANGES_REQUESTED** [review]

**Pull request closed without merging**